handler.fish

```diff
@@ -1,24 +1,14 @@
-function __fish_command_not_found_on_interactive --on-event fish_prompt
```

Comment:
Searching around GitHub and Google suggests me that this is only used by Homebrew. I don't know its original purpose, but looks unnecessary now.